Train to be a teacher (#2474)
DFE-Digital/get-into-teaching-app: app/views/content/help-and-advice.md

- find the right courses to apply for
- make a successful application
- prepare for your interview

<a href="https://adviser-getintoteaching.education.gov.uk/" class="button">Get your dedicated adviser</a>

##Get personalised guidance in your inbox

We can help you get into the classroom with guidance tailored to you, including how to:

- learn more about what teaching is really like
- find funding to train
- get one-to-one support
- read advice from trainees or early career teachers
- write a successful application

Just answer a few questions about your current situation and what you would like to teach and we will keep you up to date with the guidance and insights relevant to you.

<a href="/mailinglist/signup/name" class="button">Sign up for personalised emails</a>

##Have a chat

If you would like to talk to someone now to find out more information about all aspects of teacher training, from working out what qualifications you need to how to apply for funding, you can chat online or call us on 0800 389 2500 and we will be able to help you.

Our chat and telephone services are available Monday to Friday between 8:30am and 5:30pm.

> <a href="#" class="button">Chat online</a>

Reviewer: @peteryates Does this chat button on the Help and advice page have the usual chat functionality that it is only surfaced during hours when chat is available, otherwise there is a closed message?

Plus does it have the email fallback for non-JS users?

Author: Didn't even realise we had a chat online button here. It has none of that currently (just a link) - I think we can drop the ChatComponent in here which will have all that functionality. Will check.

title_paragraph: We’re here to answer your questions and provide advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
calls_to_action:
  chat:
    name: chat
    arguments: {}
---

## Attend an event

Our events are a great way to meet teachers and other experts to talk about any concerns or questions you have about becoming a teacher.

### Train to Teach events

Train to Teach events are run by the Department for Education (DfE) and are free to attend. You'll be able to:

- put your questions to expert advisers, teachers and training providers
- chat with current teachers
- find out what a career in teaching is really like
- watch presentations which provide detailed guidance on how to get into teaching, the application process and funding your training

### Online question and answer sessions

There are online, text-based sessions where you can ask a panel of specialists the questions that matter to you. These sessions, run by DfE, cover a range of different topics so you can post specific questions and receive advice tailored to your particular circumstances.

### School and university events

> You could also attend one of the School and University events, either online or in person, and hear directly from teacher training providers about the courses they offer and how to apply.

Reviewer: ```suggestion
You could also attend a school and university event, either online or in person, and hear directly from teacher training providers about the courses they offer and how to apply.
```

@EllieNodder Suggested change from the Events team for you to consider

NB once we have the new events design live, these events will be called 'Training provider events' rather than 'School and university events'

<a href="/events" class="button">Search for an event</a>

## Get a teacher training adviser

Teacher training advisers provide free, one-to-one support by phone, text or email wherever you are in your journey into teaching. They are experienced, former teachers and able to answer any questions you have. You will have your own dedicated adviser to support you at every step as you make important decisions about your future.

They can help you to:

- get classroom experience
- find out what teaching is really like
- understand more about teacher training courses
- understand the funding and support available to you
- find the right courses to apply for
- make a successful application
- prepare for your interview

<a href="https://adviser-getintoteaching.education.gov.uk/" class="button">Get your dedicated adviser</a>

## Have a chat

> If you would like to talk to someone now to find out more information about all aspects of teacher training, from working out what qualifications you need to how to apply for funding, you can chat online or call us on 0800 389 2500 and we will be able to help you.

Reviewer: @gazcarr @EllieNodder 

Feedback from Steph in Candidate Experience team - should we make the phone number more prominent (as it is in the Talk to us footer section)?

On mobile it is a link anyway - just whether you think it is worth doing anything that caters for desktop traffic (your call)

Reviewer 2: ![image](https://user-images.githubusercontent.com/94541493/166424060-3d3b9058-7330-4eea-a9fa-92b8b9017062.png)
@gazcarr @gemmadallmandfe we could put the phone number on a new line in bold like this?

Reviewer: @EllieNodder That looks good to me but I will defer to @gazcarr !

  Get help and advice on getting into teaching by talking to a dedicated adviser, going to a teacher training event, or registering for relevant email updates.
navigation: 50
image: "media/images/content/hero-images/0025.jpg"

> title-paragraph: We’re here to answer your questions and provide advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.

Author: ```suggestion
title_paragraph: We’re here to answer your questions and provide advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
```
